src/services/component-service.ts

```diff
@@ -64,27 +67,30 @@ export class ComponentService {
     }))
   }
 
+  public static getComponentPath(component: Component<ComponentType>): string {
+    const { name, type } = component
+    const componentPath =
+      type === ComponentType.MICROSERVICE
```

Comment:
I prefer using a switch here as we may have other components soon (EPC)

Reply:
@alepintus Inside `base-build.ts` there is a similar logic (in `buildExecutionOptions()` method). We can use this new method also there. Thanks.

Reply:
done